grammars/less.cson

```diff
@@ -300,85 +327,183 @@
+  'property_name_svg':
+    'captures':
+      '1':
+        'name': 'support.type.property-name.svg.css'
+    'match': '(?<![-a-z])(writing-mode|text-anchor|stroke-width|stroke-opacity|stroke-miterlimit|stroke-linejoin|stroke-linecap|stroke-dashoffset|stroke-dasharray|stroke|stop-opacity|stop-color|shape-rendering|marker-start|marker-mid|marker-end|lighting-color	|kerning|image-rendering|glyph-orientation-vertical|glyph-orientation-horizontal|flood-opacity|flood-color|fill-rule|fill-opacity|fill|enable-background|color-rendering|color-interpolation-filters|color-interpolation|clip-rule|clip-path)(?=\\s*:)'
```

Comment:
I fixed the whitespace, but I'm not sure about the `-`. What should the effect be? There aren't any specs for it and I'm not sure what it should accomplish in the first place. It was like that so I kept it.